nautobot_device_lifecycle_mgmt/models.py

```diff
@@ -291,11 +291,17 @@ class ValidatedSoftwareLCM(PrimaryModel):
     old_software = models.ForeignKey(
         to="SoftwareLCM", on_delete=models.SET_NULL, verbose_name="Software Version", blank=True, null=True
     )
-    devices = models.ManyToManyField(to="dcim.Device", related_name="+", blank=True)
-    device_types = models.ManyToManyField(to="dcim.DeviceType", related_name="+", blank=True)
-    device_roles = models.ManyToManyField(to="extras.Role", related_name="+", blank=True)
-    inventory_items = models.ManyToManyField(to="dcim.InventoryItem", related_name="+", blank=True)
-    object_tags = models.ManyToManyField(to="extras.Tag", related_name="+", blank=True)
+    devices = models.ManyToManyField(to="dcim.Device", related_name="validatedsoftware_devices_tab", blank=True)
+    device_types = models.ManyToManyField(
+        to="dcim.DeviceType", related_name="validatedsoftware_device_types_tab", blank=True
```

Comment:
Was this related name used for backward compatibility? Per our convention I think this should be `related_name="validated_software_lcms"` for all of these.